QuickOSM/test/test_preset_parser.py

```diff
@@ -0,0 +1,176 @@
+import xml.dom.minidom as xml
+
+from qgis.testing import unittest
```

Comment:
```suggestion
import unittest
```
This should make it working for you in Pycharm ?